OloEngine/src/OloEngine/Renderer/ReflectionProbeBaker.cpp

```
@@ -0,0 +1,180 @@
+#include "OloEngine/Scene/Scene.h"
+
+#include <glad/gl.h>
+#include <glm/gtc/matrix_transform.hpp>
+
+#include <algorithm>
+#include <limits>
+#include <vector>
+
+namespace OloEngine
+{
+    i32 SelectDominantReflectionProbe(const glm::vec3& cameraPosition,
```

Cppcheck (2.20.0)

[style] The function 'SelectDominantReflectionProbe' is never used.

(unusedFunction)